static/styles/zulip.scss

```diff
@@ -284,7 +284,9 @@ p.n-margin {
 .app {
     width: 100%;
     height: 100%;
-    overflow-y: scroll;
+    // We don't want to show scroll bar unless,
```

Comment:
@timabbott don't merge this yet, It throws off the carefully crafted alignments in various places.

Reply:
Removed this commit, I will come back to this at some other time maybe.